src/ipc.ts

```diff
@@ -70,18 +70,21 @@ export default class IPC {
     } catch (e) { }
   }
 
-  private async tab(): Promise<number> {
+  private async tab(): Promise<any> {
```

Comment:
is this `number | undefined` now?

Reply:
That's right. I added this to prevent a bunch of errors from being thrown in the console when we're in a `chrome://` tab (which doesn't have an id).